### PR TITLE
Feature/add table component

### PR DESCRIPTION
**Pull request by Jeffrey-Zutt**
## Optional sticky right column for smaller screens.
sticky-column

## Loading state
loading state

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,9 @@
     "lint": "eslint --cache --ext .ts,.tsx src",
     "lint:fix": "npm run lint -- --fix",
     "generate:swagger-schema": "dtsgen -o ./src/__generated__/apiSchema.d.ts -n API --url http://localhost:8000/api/v1/schema/",
-    "generate:form-scaffolds": "ts-node scripts/generateFormScaffolds.ts"
+    "generate:form-scaffolds": "ts-node scripts/generateFormScaffolds.ts",
+    "storybook": "start-storybook -p 9009 -s public",
+    "build-storybook": "build-storybook -s public"
```
Comment (gilleswittenberg) [inline package.json:41]: `storybook:build`

File: src/setupTests.ts [hunk 2]
```diff
@@ -3,3 +3,7 @@
 // expect(element).toHaveTextContent(/react/i)
```
Comment (gilleswittenberg) [inline src/setupTests.ts:3]: Should be removed?

File: .storybook/config.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+const Wrap = styled.div`
+  padding: ${ themeSpacing(5) };
+`
+
+// automatically import all files ending in *.stories.js
+const req = require.context("../src", true, /\.stories\.(tsx)$/)
+
+const extendedTheme = {
+  globalStyle: `
+  `,
```
Comment (gilleswittenberg) [inline .storybook/config.js:18]: Can this be an empty string `""`. Or do we need the linebreak somehow?

File: package.json (hunk 1 above)
Comment (Jeffrey-Zutt) [reply package.json:41]: Ah yeah, you're right. Storybook adds it like this by default. 
Will change

File: src/setupTests.ts (hunk 2 above)
Comment (Jeffrey-Zutt) [reply src/setupTests.ts:3]: I'd rather keep it there. It's a comment added by create-react-app.
I think its more clear to leave it in :-)

File: src/app/features/shared/components/molecules/Table/Table.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+const columns = [
+  "ID",
+  "First name",
+  "Surname",
+  "Email",
+  "Gender",
+  "Ip",
+  <>&nbsp;</>
```
Comment (gilleswittenberg) [inline src/app/features/shared/components/molecules/Table/Table.stories.tsx:20]: Maybe we can use `undefined` or something more abstract to allow for columns without a header? This seems like a hack to me at first sight.

File: .storybook/config.js (hunk 3 above)
Comment (Jeffrey-Zutt) [reply .storybook/config.js:18]: I'll change it

File: src/app/features/shared/components/molecules/Table/Table.stories.tsx (hunk 4 above)
Comment (Jeffrey-Zutt) [reply src/app/features/shared/components/molecules/Table/Table.stories.tsx:20]: Yeah, that might be better

**Pull request merged**